src/components/ui/Button.tsx

```diff
@@ -3,7 +3,9 @@ import classnames from "classnames";
 import border from "assets/ui/panel/light_border.png";
 
 interface Props {
-  onClick?: () => void;
+  onClick?:
```

Comment:
I would suggest removing the second option `(() => void)` since every `onClick` will have the `event` parameter passed through.

Parameters are optional to be implemented in the parent components so they do not need to use the `event` parameter, they can leave it blank :)

Reply:
Yep. Just wanted to be sure :) as Button is widely used.
PS: Tho I've tested manually every button, all r working fine no side effects